ee/clickhouse/queries/clickhouse_sessions.py

```diff
@@ -48,6 +50,35 @@ def calculate_list(self, filter: Filter, team: Team, limit: int, offset: int):
 
         return result
 
+    def _get_best_timerange(self, filter: Filter, team: Team):
+        count = 0
+        # so parse_timestamps will retain hours and minutes
+        filter.interval = "hour"
+
+        for i in range(1, 8):
```

Comment:
we can maybe do this another way where we query for 100 distincts in ordered events and then just take the timestamp of the earliest one